back/src/api/common/services/alcohol/alcoholTaxCalculator.ts

```
@@ -98,30 +104,56 @@
       const liters = products.reduce((sum, p) => sum + (p.taxableValue ?? 0), 0);
-      let excise = rates.exciseRate * liters;
-      const css = rates.cssRate * liters;
+      let excise = 0;
 
-      // Pour la bière et le vin, si l'accise arrondie est < 1€, on met à 0
-      if (['beer', 'wine'].includes(mappedType)) {
-        excise = Math.round(excise) < rates.minimumThreshold ? 0 : Math.round(excise);
+      // Calcul des accises selon le type d'alcool
+      if (mappedType === 'beer' && rates.exciseRate) {
+        // Pour la bière : taux par degré
+        const averageDegree = this.getAverageAlcoholDegree(products) || 8; // 8° par défaut si non spécifié
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)